gui.js

```diff
@@ -2909,6 +2909,17 @@ var CGUI = function()
       }
     }
   };
+	
+	var presetIgnoreLetters = function (e)
```

Comment:
You seem to be using the wrong indentation (tabs instead of spaces?). Please fix.